### ISSUE TITLE
Long variable value breaks Variable Preview layout

### ISSUE THREAD

**Issue by flin-8** [OctopusDeploy/Issues#5706]
# The bug

When a variable value is very long, it expands the size of the columns, pushing the variable values to the right.

![image](https://user-images.githubusercontent.com/2888279/61337389-7c82ce80-a878-11e9-989c-772d3214fa87.png)

## What I expected to happen

The value should be truncated like in the variable editor

![image](https://user-images.githubusercontent.com/2888279/61337443-b227b780-a878-11e9-8204-23864c3e795d.png)

## Steps to reproduce

1. Go to project variables
2. Add a variable with a really long value
3. Go to variable preview

## Affected versions

**Octopus Server: 2019.3 - 2019.7** 

## Links
https://help.octopus.com/t/2019-6-3-variable-preview-view-is-broken/23526/2

**Comment by matt-richardson** [comment]
This also happens on the "All Variables" page

**Comment by verzada** [comment]
@matt-richardson Yep I just saw it too today on the 2019.6.4 LTS version as well.

**Comment by KennethBates** [comment]
We should probably also consider if truncating long variable values is the best general way to display them, as it makes getting the full value from the preview harder than it should be.

This input was raised here (private link) https://secure.helpscout.net/conversation/940723612/49220?folderId=954335

**Comment by verzada** [comment]
The Variables Preview works in Internet Explorer 11 - so that is a workaround for now.

**Comment by KennethBates** [comment]
We should also make sure to address this report of display issue when a user has restricted access to view variables: https://secure.helpscout.net/conversation/953029222/50018?folderId=954335 (private link)

**Comment by Justin-Walsh** [comment]
Another report (Internal link) - https://secure.helpscout.net/conversation/1033514251/54508?folderId=2845821

**Comment by KennethBates** [comment]
Report of "All" variables page with long value: https://secure.helpscout.net/conversation/1061821942/56087/ (private link)

**Comment by thedewi** [comment]
Another confirmed report on project's Library Variable Sets page: https://help.octopus.com/t/bug-viewing-project-variables-library-sets-table-expands-to-more-than-10x-the-window-width/24590/2

**Comment by donnybell** [comment]
Another report: https://help.octopus.com/t/variable-preview-not-usable/26879

**Comment by verzada** [comment]
Any efforts looking into making long values viable?

**Comment by patrick-smergut-octopus** [comment]
Another report here: https://help.octopus.com/t/tidying-up-the-all-and-variable-preview-pages/27082

**Comment by verzada** [comment]
Any thoughts about fixing the issue?

**Comment by rpresser** [comment]
In current versions, the length is truncated but there is a tooltip with the full value including its internal line breaks. So that works for us.
